apps/web/src/routes/_view/index.tsx

```
@@ -12,51 +13,268 @@ export const Route = createFileRoute("/_view/")({
+
+                {/* CTAs */}
+                <div className="flex flex-col sm:flex-row gap-4 items-center">
                   <DownloadButton />
-                  <p className="text-neutral-500 self-center">
+                  <p className="text-neutral-500">
                     Free and{" "}
                     <a
-                      className="decoration-dotted underline hover:text-blue-600 transition-all"
+                      className="decoration-dotted underline hover:text-stone-600 transition-all"
                       href="https://github.com/fastrepl/hyprnote"
                       target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)